### PR TITLE
Add aspnet v5 test

### PR DESCRIPTION
**Pull request by asvrada**

### CODE REVIEW

File: tests/aspnetcore5/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    // OpenEnclave specific values
+
+    "Debug": 1,
+    "ProductID": 1,
+    "SecurityVersion": 1,
+
+    // Mystikos specific values
+
+    "MemorySize": "8g",
```
Comment (anakrish) [inline tests/aspnetcore5/config.json:10]: Do we need 8g for running the tests? The other aspnetcore uses only 2G.

File: tests/aspnetcore5/testhost.all [hunk 2]
```diff
@@ -0,0 +1,140 @@
+./aspnetcore/artifacts/bin/AuthSamples.FunctionalTests/Debug/net5.0/testhost.dll
```
Comment (anakrish) [inline tests/aspnetcore5/testhost.all:1]: Do we need this file?

File: tests/aspnetcore5/unit-tests.failed [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Cryptography.Internal.Tests/Debug/net5.0/Microsoft.AspNetCore.Cryptography.Internal.Tests.dll takes forever
+# /aspnetcore/artifacts/bin/Microsoft.AspNetCore.DataProtection.Abstractions.Tests/Debug/net5.0/Microsoft.AspNetCore.DataProtection.Abstractions.Tests.dll
+# /aspnetcore/artifacts/bin/Microsoft.AspNetCore.DataProtection.Tests/Debug/net5.0/Microsoft.AspNetCore.DataProtection.Tests.dll takes forever
+# /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Http.Connections.Tests/Debug/net5.0/Microsoft.AspNetCore.Http.Connections.Tests.dll takes forever
+# /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Identity.FunctionalTests/Debug/net5.0/Microsoft.AspNetCore.Identity.FunctionalTests.dll would crash
+# /aspnetcore/artifacts/bin/Microsoft.AspNetCore.JsonPatch.Tests/Debug/net5.0/Microsoft.AspNetCore.JsonPatch.Tests.dll would crash
+# /aspnetcore/artifacts/bin/Microsoft.AspNetCore.Server.Kestrel.Core.Tests/Debug/net5.0/Microsoft.AspNetCore.Server.Kestrel.Core.Tests.dll would crash/exit the runner
```
Comment (anakrish) [inline tests/aspnetcore5/unit-tests.failed:9]: Why are some tests commented out and other not commented out?

File: tests/Makefile [hunk 4]
```diff
@@ -133,6 +133,7 @@ DIRS += fsflags
 ifdef MYST_NIGHTLY_TEST
 DIRS += glibc
 DIRS += aspnetcore
```
Comment (jxyang) [inline tests/Makefile:135]: Please rename the original one to `aspnetcore6`

File: tests/Makefile [hunk 5]
```diff
@@ -132,7 +132,8 @@ DIRS += fsflags
 
 ifdef MYST_NIGHTLY_TEST
 DIRS += glibc
-DIRS += aspnetcore
+DIRS += aspnetcore5
+DIRS += aspnetcore6
```
Comment (jxyang) [inline tests/Makefile:136]: I think we should put both aspnetcore5 and aspnetcore6 in the PR pipeline as they are taking much shorter time than before.

Comment (asvrada) [reply tests/Makefile]: Running these two tests take about 20-30 minutes, and the [PR pipeline timed out](https://openenclave.visualstudio.com/ACC-Services/_build/results?buildId=13161&view=logs&j=ab477ad6-9781-5b54-82ca-a0141b26726e&t=a1e294d4-8895-5a45-41ff-022cede0621b). Should move back to nightly pipeline

Comment (jxyang) [reply tests/Makefile]: Sure

**Review by jxyang — APPROVED** [review]

**Pull request merged**